package.json

```diff
@@ -27,6 +27,7 @@
   },
   "scripts": {
     "start": "webpack-dev-server --hot --inline --config webpack.dev.js",
+    "start-ci": "NO_ELM_DEBUGGER=true npm run start",
```

Comment:
cool trick, but this should be done in a different PR, or at least a different commit

Reply:
Fair enough, I'll add a separate commit